mods/farming/api.lua

```diff
@@ -176,6 +177,7 @@ farming.place_seed = function(itemstack, placer, pointed_thing, plantname)
 	end
 
 	-- add the node and remove 1 item from the itemstack
+	minetest.log("action", placer:get_player_name() .. " places node " .. " at " .. minetest.pos_to_string(place_to))
```

Comment:
there is `player_name` already
also, you are missing the node name

Reply:
Oh yeah xD I removed it while testing. How stupid of me not to remember to put it back. I'll do that now.

Reply:
Done!